packages/react-examples/package.json

```diff
@@ -52,6 +52,7 @@
     "@fluentui/react-focus": "^8.1.4",
     "@fluentui/react-hooks": "^8.2.2",
     "@fluentui/react-icons-mdl2": "^1.1.3",
+    "@fluentui/react-image": "^9.0.0-alpha.45",
```

Comment:
hmm bad rebase ? react-image was migrated to new DX, thus it should not figure within react-examples